diff --git a/clients/android/app/src/main/java/app/lockbook/util/WorkspaceView.kt b/clients/android/app/src/main/java/app/lockbook/util/WorkspaceView.kt
index b8315c5b..67790b05 100644
--- a/clients/android/app/src/main/java/app/lockbook/util/WorkspaceView.kt
+++ b/clients/android/app/src/main/java/app/lockbook/util/WorkspaceView.kt
@@ -89,10 +89,6 @@ class WorkspaceView(context: Context, val model: WorkspaceViewModel) : SurfaceVi
         nativeLock.withLock { }
     }
 
-    private fun adjustTouchPoint(axis: Float): Float {
-        return axis / context.resources.displayMetrics.scaledDensity
-    }
-
     @SuppressLint("ClickableViewAccessibility")
     override fun onTouchEvent(event: MotionEvent?): Boolean {
         if (event != null) {
@@ -288,8 +284,8 @@ class WorkspaceView(context: Context, val model: WorkspaceViewModel) : SurfaceVi
                     Workspace.touchesBegin(
                         WGPU_OBJ,
                         pointerId,
-                        adjustTouchPoint(event.getX(i)),
-                        adjustTouchPoint(event.getY(i) + touchOffsetY),
+                        event.getX(i),
+                        event.getY(i) + touchOffsetY,
                         0.0f
                     )
                 }
@@ -298,8 +294,8 @@ class WorkspaceView(context: Context, val model: WorkspaceViewModel) : SurfaceVi
                     Workspace.touchesMoved(
                         WGPU_OBJ,
                         pointerId,
-                        adjustTouchPoint(event.getX(i)),
-                        adjustTouchPoint(event.getY(i) + touchOffsetY),
+                        event.getX(i),
+                        event.getY(i) + touchOffsetY,
                         0.0f
                     )
                 }
@@ -308,8 +304,8 @@ class WorkspaceView(context: Context, val model: WorkspaceViewModel) : SurfaceVi
                     Workspace.touchesEnded(
                         WGPU_OBJ,
                         pointerId,
-                        adjustTouchPoint(event.getX(i)),
-                        adjustTouchPoint(event.getY(i) + touchOffsetY),
+                        event.getX(i),
+                        event.getY(i) + touchOffsetY,
                         0.0f
                     )
                 }
@@ -318,8 +314,8 @@ class WorkspaceView(context: Context, val model: WorkspaceViewModel) : SurfaceVi
                     Workspace.touchesCancelled(
                         WGPU_OBJ,
                         pointerId,
-                        adjustTouchPoint(event.getX(i)),
-                        adjustTouchPoint(event.getY(i) + touchOffsetY),
+                        event.getX(i),
+                        event.getY(i) + touchOffsetY,
                         0.0f
                     )
                 }
